hack/snapshot_to_image_list.sh

```
@@ -1,253 +1,184 @@
+		fi
 	fi
-fi
 
-if [ -z "${POSTGRES_IMAGE}" ] || [ "${POSTGRES_IMAGE}" == "null" ]; then
-	if [ -f "${OUTPUT_FILE}" ]; then
-		POSTGRES_IMAGE=$(jq -r '.[] | select(.name == "lightspeed-postgresql") | .image' "${OUTPUT_FILE}")
+	if [ "${name}" = "lightspeed-postgresql" ]; then
+		if [ -z "${image}" ] || [ "${image}" = "null" ]; then
+			image=$(postgres_default_image)
+		fi
+		image=$(sed 's|quay\.io.*/lightspeed-postgresql|registry.redhat.io/rhel9/postgresql-16|g' <<<"${image}")
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)